Mathlib/AlgebraicGeometry/Morphisms/Constructors.lean

```diff
@@ -60,6 +60,7 @@ instance AffineTargetMorphismProperty.diagonal_respectsIso (P : AffineTargetMorp
     rw [pullback.mapDesc_comp, P.cancel_right_of_respectsIso]
     apply H
 
+attribute [-ext] Scheme.Hom.ext in
```

Comment:
Is this an indication that the `@[ext]` on `Scheme.Hom.ext` should be dropped?

Reply:
I think when the user invokes `ext` when the goal is the equality of two morphisms, `Scheme.Hom.ext` is probably what one would expect, and we should allow this behavior. In this case I could instead change the `ext` to an `ext1` (which is what I should do instead) so I don't think this `ext`-lemma is that intrusive.